### DISCUSSION TITLE
Mobile app for doing translations?

### DISCUSSION THREAD

**Discussion by sanketgarade** [WeblateOrg/weblate#8950] [Ideas] [answered]
Are there any plans to create a mobile app for weblate?

I am aware that translation can also be done via a browser from a mobile, but i think an app will be beneficial if made such that a lot of strings (and other data) can be downloaded at one time, edited on the app offline and in a faster way, and then be uploaded in one go.

I often edit on the go, translating a few strings at a time whenever i get time, instead of sitting in front of a laptop for a long duration. For such a use case, I think the app will be useful.

Thanks.

**Comment by nijel** [comment, marked as answer]
We currently have no plans to develop such an app. We're happy to discuss API extensions to make it possible. See also https://github.com/WeblateOrg/weblate/issues/1066

**Comment by infureal** [comment]
I'm parttime Flutter dev and I like to participate

**Reply by trymeouteh** [reply]
A desktop and mobile app built with Flutter/Dart or Tauri would be great!

https://github.com/WeblateOrg/weblate/discussions/11260

**Comment by trymeouteh** [comment]
https://github.com/WeblateOrg/weblate/discussions/11260
